src/api/custom-resolvers/index.ts

```diff
@@ -18,4 +19,5 @@ export {
   addAndGetOTP,
   deleteApplicationByReferenceNumber,
   submitApplication,
+  sendEmailInsuranceFeedback,
```

Comment:
Could this be `sendEmailFeedback` unless there are differences between quote and insurance?

Reply:
Most likely will be different templates as different fields as more fields on the feedback form for quote